deps/spidershim/src/v8arraybuffer.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright Mozilla Foundation. All rights reserved.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+
+#include "v8.h"
+#include "v8conversions.h"
+#include "v8local.h"
+#include "jsapi.h"
+#include "jsfriendapi.h"
+#include "js/Conversions.h"
+
+static_assert(sizeof(v8::Value) == sizeof(JS::Value),
+              "v8::Value and JS::Value must be binary compatible");
+
+namespace v8 {
+
+Local<ArrayBuffer>
+ArrayBuffer::New(Isolate* isolate, size_t size)
+{
+	if (!isolate) {
```

Comment:
OMG tabs :((

I'll need to clang-format immediately after merging your commit, I guess!